src/pde.cpp

```diff
@@ -442,11 +537,10 @@ dolfin::Function PDE::_convert_EigenVector_to_Function (
 ) {
   dolfin::Function fn(_function_space);
 
-  if (eigen_vector.size() != _solution_function->vector()->size()) {
-    printf("Cannot convert EigenVector to Function...\n");
-    printf("\tincompatible dimensions!\n");
-  }
-
+  // if (eigen_vector.size() != _solution_function->vector()->size()) {
```

Comment:
Are you ok removing this line? because in the case of PNP+Stokes '_solution_function" does not exists.